script/bulk_import/vbulletin.rb

```diff
@@ -5,6 +5,44 @@
 class BulkImport::VBulletin < BulkImport::Base
 
   SUSPENDED_TILL ||= Date.new(3000, 1, 1)
+  CHARSET_MAP = {
+    "armscii8" => nil,
```

Comment:
Old version of mysql2:
http://www.rubydoc.info/gems/mysql2/0.3.11/Mysql2/Client